dspace-api/src/main/java/org/dspace/statistics/util/StatisticsImporterElasticSearch.java

```diff
@@ -205,7 +206,7 @@ private void load(String filename, Context context, boolean verbose)
                         }
                         continue;
                     }
-                } catch (Exception e) {
+                } catch (GeoIp2Exception | IOException e) {
```

Comment:
also, should we be using logical || OR instead of bitwise | OR?

Reply:
I'll look into handling the AddressNotFoundException separately.  Thanks!

The bar operator is overloaded here.  It means "any of these exceptions".  The catch condition is not an expression.

Reply:
AddressNotFoundException is a subclass of GeoIp2Exception, so that catch clause should catch it and discard it.  Do you mean that you want to log it instead of just ignoring it?

Reply:
Ah yes, you're absolutely right, I'd not thought of it as being a subclass and so didn't notice that it essentially is already being logged:
`log.error("Unable to get location of request", e);`

If we're OK with the full stack trace of `e` getting logged then no problem... I just thought since this is probably something that will happen a lot with any private address space browsing to a repository, we might want to log `e.getMessage()` for these particular `AddressNotFound`  instances?

Reply:
StatisticsImporterElasticSearch just discards the exception without logging.  But SolrLoggerServiceImpl logs it, and yes, the trace is too much.  Do we need *any* of these exceptions traced?  It would be simple to just have them all log e.getMessage() rather than e.

That change should be ported back to master as well.